packages/default/scss/input/_layout.scss

```diff
@@ -50,13 +50,14 @@
     }
 
 
-    // Input
+    // Input and Textarea
     .k-input {}
-    input.k-input {
+    input.k-input,
+    textarea.k-textarea {
         padding: $kendo-input-padding-y $kendo-input-padding-x;
     }
 
-
+    
```

Comment:
there is trailing space on this line, which means you don't have the correct setup. go to develop, pull and run `npm run setup`. after that, rebase this branch and fix the lint errors.